spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/NettyWriteResponseFilter.java

```diff
@@ -75,7 +75,13 @@ public int getOrder() {
 					.retain() //TODO: needed?
 					.map(factory::wrap);
 
-			MediaType contentType = response.getHeaders().getContentType();
+			MediaType contentType;
+			try {
+				contentType = response.getHeaders().getContentType();
+			} catch (Exception e) {
+				log.trace("invalid media type", e);
+				return response.writeWith(body);
```

Comment:
I think we can just log the exception and remove the return.  `isStreamingMediaType` should handle `null`.